packages/extension/src/companion/CompanionPermission.tsx

```diff
@@ -38,7 +38,7 @@ const CompanionPermissionComponent = (
         </p>
         <Button
           className="mt-1 w-[12.5rem] btn btn-primary"
-          onClick={requestContentScripts}
+          onClick={() => requestContentScripts()}
```

Comment:
This change shouldn't be needed

Reply:
the `requestContentScripts` function now accepts a `skipRedirect` parameter as an argument, so passing it directly causes a TS error since `onClick` expects an event instance to be the argument of the function being passed in